client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -53,12 +54,43 @@ const EditCardDetails = React.createClass( {
 		'postalCode'
 	],
 
+	componentWillReceiveProps( nextProps ) {
+		// Updates the form once with the stored credit card data as soon as they are available
+		if ( nextProps.card && ( ! this.props.card || ( nextProps.card.id !== this.props.card.id ) ) ) {
+			this.setState( {
+				form: formState.initializeFields( this.state.form, this.mergeCard( nextProps.card ) )
+			} );
+		}
+	},
+
+	/**
+	 * Merges the specified card object returned by the StoredCards store into a new object with only properties that
+	 * should be used to prefill the credit card form, and with keys matching the corresponding field names.
+	 *
+	 * @param card
+	 * @param fields
+	 */
+	mergeCard( card, fields: {} ) {
```

Comment:
Yes, it looks like a default param. A few lines above it's called without 2nd param, so that makes perfect sense.